testing/matrix-sdk-integration-testing/src/tests/e2ee/shared_history.rs

```diff
@@ -35,31 +36,31 @@ async fn test_history_share_on_invite() -> Result<()> {
         .build()
         .await?;
 
-    let sync_service_span = tracing::info_span!(parent: &alice_span, "sync_service");
-    let alice_sync_service = Arc::new(
-        SyncService::builder(alice.clone())
-            .with_parent_span(sync_service_span)
-            .build()
-            .await
-            .expect("Could not build alice sync service"),
+    let bob = SyncTokenAwareClient::new(
+        TestClientBuilder::new("bob").encryption_settings(encryption_settings).build().await?,
     );
```

Comment:
feels a bit weird to me that we have this bit of bob setup in the middle of all the alice setup, but not a strong opinion.